src/main/java/com/dawsonsystems/session/MongoManager.java

```diff
@@ -494,7 +420,7 @@ public void removePropertyChangeListener(PropertyChangeListener propertyChangeLi
   public void processExpires() {
     BasicDBObject query = new BasicDBObject();
 
-    long olderThan = System.currentTimeMillis() - (getMaxInactiveInterval() * 1000);
+    long olderThan = System.currentTimeMillis() - (getSessionMaxAliveTime() * 1000);
 
     log.fine("Looking for sessions less than for expiry in Mongo : " + olderThan);
```

Comment:
Use https://docs.oracle.com/en/java/javase/11/docs/api/java.logging/java/util/logging/Logger.html#isLoggable(java.util.logging.Level) or https://docs.oracle.com/en/java/javase/11/docs/api/java.logging/java/util/logging/Logger.html#info(java.util.function.Supplier) to avoid string concatenation in hot path when FINE is not enabled.